runtime/parachains/src/configuration.rs

```diff
@@ -1140,36 +1164,84 @@ impl<T: Config> Pallet<T> {
 			.last()
 			.map(|&(_, ref config)| config.clone())
 			.unwrap_or_else(Self::config);
+		let base_config_consistent = base_config.check_consistency().is_ok();
 
 		// Now, we need to decide what the new configuration should be.
+		// We also move the `base_config` to `new_config` to empahsize that the base config was
+		// destroyed by the `updater`.
 		updater(&mut base_config);
+		let new_config = base_config;
+
+		if <Self as Store>::BypassConsistencyCheck::get() {
+			// This will emit a warning each configuration update if the consistency check is
+			// bypassed. This is an attempt to make sure the bypass is not accidentally left on.
+			log::warn!(
```

Comment:
FYI: Logging is disabled by default for the on chain wasm. While we run nodes with logging enabled, you can not bet on this.

Reply:
yeah, I thought this is better than nothing. Do you have a better suggestion?

Reply:
If it is important, you could send an event. However, I just wanted to remind you.